File: c language/question 31.c
// 31. Write a program to repeat a string a specified number of times.Example: Input: ("hello", 3), Output: "hellohellohello".
#include <stdio.h>
#include <string.h>
int main(){
    char input[100];
    printf("enter input");
    scanf("%s",&input);
    int num;
    printf("enter number of times");
    scanf("%d",&num);
    char temp[100]="";
    for(int i=0;i<num;i++){
        strcat(temp,input);
    }
    printf("%s",temp);
    return 0;
}

File: Java script/question 31.js
// 31. Write a program to repeat a string a specified number of times.Example: Input: ("hello", 3), Output: "hellohellohello".
var string =prompt("enter input")
var int =prompt("enter the value how many times you want to repeat")
var temp=""
for(i=0;i<int;i++){
  temp=temp+string
}
console.log(temp)